### DISCUSSION TITLE
Octodash files for screens

### DISCUSSION THREAD

**Discussion by MrDLoomis** [UnchartedBull/OctoDash#1974] [Q&A] [answered]
Is it possible to edit and modify the screens in Octodash?  For example, I'd like to move the Cancel button to the right side (currently on left), and move the Adjust button to the left side.  

If this is possible, where are the files located?  I've looked through the entire tree and can't find them.

Thanks!

**Comment by UnchartedBull** [comment, marked as answer]
https://github.com/UnchartedBull/OctoDash/blob/main/src/app/print-control/print-control.component.html

**Reply by MrDLoomis** [reply]
Where does this file live?  Or do I need to add it somewhere?  I've looked through the tree on the pi I'm using for Octodash, and can't seem to find it anywhere.  Thanks.

**Reply by UnchartedBull** [reply]
Well it doesn't exist like that in the Pi since everything gets squashed into a single file inside the app package. You'll need to change that file and then build the app on your own as described in `CONTRIBUTING.md`
